pkg/common/log/options.go

```diff
@@ -9,38 +9,46 @@ import (
 )
 
 const (
+	// DefaultFormat signifies the default logrus format
 	DefaultFormat = ""
-	JSONFormat    = "JSON"
-	TextFormat    = "TEXT"
+	// JSONFormat signifies JSON logging format
+	JSONFormat = "JSON"
+	// TextFormat signifies Text logging format
+	TextFormat = "TEXT"
 )
 
 // An Option can change the Logger to apply desired configuration in NewLogger
 type Option func(*Logger) error
 
+// WithStdOut configures logger to log with a `stdout` hook
+func WithStdOut() Option {
```

Comment:
I would also be fine with providing the special string `"stdout"` or `"stderr"` and interpreting that in `WithOutputFile()`, zap does something similar https://godoc.org/go.uber.org/zap#Open

Reply:
it's slightly more hard to read I think (though could be missing something with os/io libs), as we'd have to have special handling of the constants both in `WithOutputFile` (early return, don't try to open the file) and in the Logger construction or closer (don't try to close it)

I'm ultimately fine with either, just saying why I went with this more explicit option